testsuite/integration/basic/src/test/java/org/wildfly/test/integration/microprofile/config/smallrye/app/MicroProfileConfigTestCase.java

```diff
@@ -200,9 +200,6 @@ public void testGetIntegerProperties() throws Exception {
             assertTextContainsProperty(text, "integerDefault", -42);
             assertTextContainsProperty(text, SubsystemConfigSourceTask.INTEGER_OVERRIDDEN_PROP_NAME,
                     SubsystemConfigSourceTask.INTEGER_OVERRIDDEN_PROP_VALUE);
-
-            assertTextContainsProperty(text, "intBadValue", 0);
```

Comment:
What is the reason for removing this asserts? Is it related with https://github.com/eclipse/microprofile-config/issues/446 ?

Reply:
this assert was incorrect and revealed a bug in smallrye-config.

When the config property had a default value that could not be converted to the expected type, smallrye-config set it to the "default value" of the types (e.g. `0` for `int`).

With https://github.com/smallrye/smallrye-config/issues/211, it has been fixed. The default value will be converted and this throw an exception that would prevent the deployment to be deployed.